holmes/plugins/toolsets/prometheus/prometheus.py

```diff
@@ -289,14 +289,16 @@ def __init__(self, toolset: "PrometheusToolset"):
         self._cache = None
 
     def _invoke(self, params: Any) -> StructuredToolResult:
-        if not self.toolset.config or not self.toolset.config.prometheus_url:
+        if not self.toolset.config or not self.toolset.config.get("prometheus_url"):
```

Comment:
Hey @vrutkovs I think we want to fix the config issue by https://github.com/robusta-dev/holmesgpt/pull/535. The config object should be initialized during check-prerequisite, but was skipped for status check.